src/main/java/gregtech/common/covers/CoverMachineController.java

```diff
@@ -115,7 +115,7 @@ public ModularUI createUI(EntityPlayer player) {
         updateDisplayInventory();
         return ModularUI.defaultBuilder()
             .label(10, 5, "cover.machine_controller.name")
-            .widget(new SliderWidget("cover.machine_controller.redstone", 10, 20, 156, 20, 1.0f, 16.0f,
+            .widget(new SliderWidget("cover.machine_controller.redstone", 10, 20, 156, 20, 1f, 15.0f,
```

Comment:
There is unnecessary change regarding 1.0f -> 1f it will not have any effect on code but for consistency sake and for making PRs as small as possible (so it's more readable).

Reply:
as you can see here :
 https://github.com/GregTechCE/GregTech/blob/59cf924550f97635e1f576bf68309adcb8bface9/src/main/java/gregtech/common/covers/CoverMachineController.java#L190
my PR should not make any breaking change by reducing redstone requirement.
I understand that maintaining overall code quality is important don't worry I'll did any required changes until it become perfect.